mvpa2/clfs/distance.py

```diff
@@ -39,12 +40,17 @@ def absmin_distance(a, b):
     return max(abs(a-b))
 
 
-def manhatten_distance(a, b):
-    """Return Manhatten distance between a and b
+def manhattan_distance(a, b):
+    """Return Manhattan distance between a and b
     """
     return sum(abs(a-b))
 
 
+@deprecated("Use correctly spelled manhattan_distance instead")
+def manhatten_distance(a, b):
+    return manhattan_distance(a, b)
```

Comment:
and how would we know that old code works and you haven't made a typo here? ;)  just include a single call within a unittest please and compare result for a good measure.  Then coverage would be happy, and I would be thrilled to merge it ;)